apps/Test App/Upload Test App.xcodeproj/project.pbxproj

```diff
@@ -466,7 +466,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Test App/Preview Content\"";
-				DEVELOPMENT_TEAM = CX6AHWLHM6;
+				DEVELOPMENT_TEAM = XX95P4Y787;
```

Comment:
question(non-blocking): Were these changes to the `DEVELOPMENT_TEAM` intentional? If no, revert?

Reply:
Good catch, this shouldn't be here, I'll revert

Reply:
Removed